public/app/features/chat/routes.tsx

```diff
@@ -0,0 +1,15 @@
+import { SafeDynamicImport } from 'app/core/components/DynamicImports/SafeDynamicImport';
+import { config } from 'app/core/config';
+import { RouteDescriptor } from 'app/core/navigation/types';
+
+const chatRoutes = [
```

Comment:
I don't think we need to start with the org wide chat page.  That feels much less useful than a commenting system within panels/dashbaords

Reply:
Uses a separate feature toggle now

Reply:
removed